js/src/components/contact-information/store-address-card.js

```diff
@@ -21,12 +21,13 @@ import TrackableLink from '.~/components/trackable-link';
 import './store-address-card.scss';
 
 /**
- * "Edit WC store address" Tracking event
+ * Triggered when store address "Edit in WooCommerce Settings" button is clicked.
+ * Before `1.5.0` it was called `edit_mc_store_address`.
  *
  * @event gla_edit_wc_store_address
  * @type {Object} TrackingEvent
```

Comment:
```suggestion
 * @type {Object}
```

Reply:
Avoiding 

`WARNING: The @type tag does not permit a description; the description will be ignored. File: store-address-card.js, line: 23`

Reply:
I removed that in https://github.com/woocommerce/google-listings-and-ads/pull/1369